ibm/mas_devops/roles/cp4d_service/defaults/main.yml

```diff
@@ -46,6 +46,7 @@ cpd_service_name: "{{ lookup('env', 'CPD_SERVICE_NAME') }}"
 # The storage class to use for this CP4D service
 cpd_service_storage_class: "{{ lookup('env', 'CPD_SERVICE_STORAGE_CLASS') }}"
 cpd_service_block_storage_class: "{{ lookup('env', 'CPD_SERVICE_BLOCK_STORAGE_CLASS') }}" # needed for WD
+cpd_service_file_storage_class: "{{ lookup('env', 'CPD_SERVICE_FILE_STORAGE_CLASS') }}" # needed for Cognos Analytics
```

Comment:
will `cpd_service_file_storage_class ` value be any different from `cpd_service_storage_class ` ? Normally `cpd_service_storage_class` is already used primarily for file storage classes so this new var should not be needed.

Reply:
All 3 of these settings were required for the CAService to be installed properly